### ISSUE TITLE
web-mode-attribute functions fail at specific point locations

### ISSUE THREAD

**Issue by dustinpaluch** [fxbois/web-mode#759]
1. `emacs -q`
2. `(package-initialize)`
3. `C-x C-f test.php RET`
4. `M-x web-mode RET`

### 

Paste this line into the buffer:
```
<a href="testing"></a>
```
**Calling `C-c C-a k` with the point at the second character of an attribute (e.g. the `r` in `href`) does not work as expected.** I get `<ahref="testing"></a>` instead of `<a ></a>`.

**Related bug:** Placing the point anywhere after the `r` in `href` causes `C-c C-a p` to move the point back to `r` instead of `h`. Calling it again will move point from `r` to `h`.
```
--- WEB-MODE DEBUG BEG ---
versions: emacs(24.5) web-mode("14.0.8")
vars: engine("php") minor("php") content-type("html") file("/Users/imacintel/Desktop/test.php")
system: window(ns) config("x86_64-apple-darwin14.5.0")
colors: fg(nil) bg(nil) 
minor modes: nil
vars:
web-mode-enable-current-column-highlight=nil 
web-mode-enable-current-element-highlight=nil 
indent-tabs-mode=t 
--- WEB-MODE DEBUG END ---
```

**Comment by fxbois** [comment]
@dustinpaluch should be better with last commit

**Comment by dustinpaluch** [comment]
The first bug is fixed. Thank you.
The related bug remains.
```
<a class="one" href="two"></a>
```
With point on the word "two", hitting `C-c C-a p` twice should move the point to the `h` in `href`, then to the `c` in `class`. 

Instead, it moves point to the `r`, and then the `h` in `href`, and it takes two additional `C-c C-a p` invocations to get to the `c`.

**Comment by fxbois** [comment]
@dustinpaluch I've pushed a fix

**Comment by dustinpaluch** [comment]
Seems to be fixed in c12be4b. Thanks!
